app/src/main/java/com/pysiak/simple/clock/ui/SimpleClockActivity.java

```diff
@@ -0,0 +1,39 @@
+package com.pysiak.simple.clock.ui;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.design.widget.TabLayout;
+import android.support.v4.app.Fragment;
+import android.support.v4.view.ViewPager;
+import android.support.v7.app.AppCompatActivity;
+import com.pysiak.simple.clock.R;
+
+import java.util.List;
+
+public class SimpleClockActivity extends AppCompatActivity implements SimpleClockView {
```

Comment:
Add Javadoc

You will have base description for in SimpleClockView, so here you need something like:
Implementation of SimpleClockView, etc